### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by sergiocortessat**
Pull request 1 To-Do-List

Here I present the To-Do-List project on javascript. Although it took me several days, I have added the specific requirements requested such as:

- Javascript ES6
- Arrow functions
- Local storage
- Babel
- Modularize folders (functional programming)
- The user can create as many projects as they wish. (The page has a preloaded project and todo item which come as preset values)
- The user can add as many to-do items as he wishes to its projects.
- The form fields allow the user to select which project they wish to save their new to-do items
- User can complete each tag by clicking over a todo- item. However, such even won't be deleted until the users decide to clear the completed tasks.
- User can delete at will each project
- User can edit at will each project to-do item and save it to the locastorage.

Note: In order to erase a task, you must first need to select the task to mark it as completed, and then you can select the button clear tasks

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,12 +1,164 @@
...
+        <h2 class="text-center mt-3">TODO ITEMS</h2>
+        <button class="add-todo w-100 btn btn-secondary m-1">
+          <i class="bi bi-plus"></i>
+        </button>
+        <div class="todo-items mt-3">
+          <ul class="todo-items-list fs-4"></ul>
+        </div>
+        <div class="buttons d-flex justify-content-evenly my-3">
+          <!-- <button class="btn btn-secondary m-1">Clear Tasks</button>
+        <button class="btn-delete btn btn-secondary m-1">Delete List</button> -->
```
Comment (bellom) [inline dist/index.html:66]: Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history. 

image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import './style.css';
+// import { format } from 'date-fns';
```
Comment (bellom) [inline src/index.js]: Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    };
+    projects.push(project);
+  }
+  addProjectsToDropdown();
+  renderProjects(projects);
+  findProjectButtons();
+  localStorage.setItem('projects', JSON.stringify(projects));
+});
+
+const submitButton = () => {
```
Comment (bellom) [inline src/index.js:81]: Let's improve this codebase, right after a user clicked submit for either for creation of the project or todo, the input field should be erased or cleared.

Example:
I want to create a new project;
In the input text, I typed Frontend, if I want to create a new project, I would have to erase the previous Frontend but I could input new text.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @sergiocortessat  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**1. Running your app locally seems to break and also not updated with the live demo link**

**2. Kindly check out my comments below;**



### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by sergiocortessat** [comment]
Pull request 2

Thanks, TSE for checking on my project. I have added the specified required changes. Making sure the local version is the same as the live version. I do recommend clearing the local storage before starting again.

Changes

- Inline code present in index.js has been removed as requested
- The project input placeholder is now cleared after every insertion of a new project as requested by TSE improvements
- I have improved the hide button which was breaking on local development. The hide button is now more intuitive. Taking over the control of the edit button as a parent node. Therefore the hide button is now dynamic to close or show all submit forms, either for edit form or new submission form.

File: dist/index.html (hunk 1 above)
Comment (addod19) [reply dist/index.html:66]: This comment is not addressed, kindly remove commented code. Use git history instead

File: src/index.js (hunk 2 above)
Comment (addod19) [reply src/index.js]: Fixed

File: src/index.js (hunk 3 above)
Comment (addod19) [reply src/index.js:81]: Clear the input field after the creation of a project or todo. Let us address this comment

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 


Hi @sergiocortessat, great job on this project, however, it seems like the comment from the previous review has not been addressed properly. Kindly check the comments below:

- I am unable to run the "npm run start" command, it returns an error. Kindly check the screenshot to appreciate the problem. Probable a babel-loader problem
re

[Optional]
- Kindly check the typo for Javascript in the PR, it is mistakenly written as "Javascriot"

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by sergiocortessat** [comment]
Pull request 3

Hello TSE. Thanks for checking my project.
I have replicated the problem on my machine and a colleague's machine. Could you please kindly run the command
 $ 'npx webpack' 

and follow the instructions. It will request for you to accept the installation of webpack-cli. As such. I suggest this on my readme, section Getting Started clone project.  This might be due to a lack of building components necessary to construct babel. I have try with npx webpack and the project builds and runs after it.

I'll be aware to know if this suggestion fixes the problem.

Screenshot

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @sergiocortessat**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**